config.example.toml

```diff
@@ -47,6 +47,12 @@
 # Run inside tmux (default: false)
 # run_in_tmux = true
 
+# Changes the way topgrade interacts with
+# the newly crated tmux session, creating the session
+# and only attaching to it if not inside tmux
+# (default: create, allowed values: "create", "create_and_switch_client")
+# tmux_session_attach_mode = "create"
```

Comment:
I am thinking about the naming, what about we call them:

1. AttachIfNotInASession
2. AlwasyAttach

I think `switch_client` is too technical, thoughts on this?

Reply:
I changed some names, also dropping `attach` infix from tmux params, to not repeat it